spec/models/user_spec.rb

```diff
@@ -71,6 +75,116 @@
     end
   end
 
+  describe "#required_credits_earned" do
+    before { user.save }
+    let!(:course1) { create :course, degree_requirement: :required_course }
+    let!(:course2) { create :course, degree_requirement: :required_course }
+    let!(:course3) { create :course, degree_requirement: :required_course }
+    let!(:course4) { create :course,
+                     degree_requirement: :distribution_requirement }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 84, 68 should be on its own line.